src/test/regress/expected/multi_extension.out

```diff
@@ -39,6 +55,23 @@ ALTER EXTENSION citus UPDATE TO '6.0-10';
 ALTER EXTENSION citus UPDATE TO '6.0-11';
 ALTER EXTENSION citus UPDATE TO '6.0-12';
 ALTER EXTENSION citus UPDATE TO '6.0-13';
+ALTER EXTENSION citus UPDATE TO '6.0-14';
+-- ensure no functions were created outside pg_catalog
+SELECT COUNT(*)
+FROM pg_depend AS pgd,
+	 pg_extension as pge,
+	 pg_proc as pgp
+WHERE pgd.classid    = 'pg_proc'::regclass AND
+	  pgd.refclassid = 'pg_extension'::regclass AND
+	  pgd.objid      = pgp.oid AND
+	  pgd.refobjid   = pge.oid AND
+	  pge.extname    = 'citus' AND
+	  pgp.pronamespace != 'pg_catalog'::regnamespace;
+ count 
```

Comment:
Hm. I don't quite see why we want to restrict this to pg_proc - shouldn't we warn about any references outside pg_catalog?